pymc/func_utils.py

```diff
@@ -50,24 +50,26 @@ def find_constrained_prior(
         Lower bound to get `mass` % of probability of `pm_dist`.
     upper : float
         Upper bound to get `mass` % of probability of `pm_dist`.
-    init_guess: Dict[str, float]
+    init_guess : str or float
```

Comment:
This is changing the type info of the parameter. It should be

```suggestion
    init_guess : dict of {str : float} 
```

Same for the other parameters. Ref: https://pandas.pydata.org/pandas-docs/stable/development/contributing_docstring.html#parameter-types